experiments/conductor/cmd/runner/runner.go

```diff
@@ -700,3 +701,40 @@ func splitMetadata(opts *RunnerOptions, branches Branches) {
 		}
 	}
 }
+
+func shouldKeepBranch(branch Branch) bool {
+	// Keep if Skip is false
+	if !branch.Skip {
+		return true
+	}
+
+	// Skip if gcloud command is empty
+	if branch.Command == "" {
+		return false
```

Comment:
Please take a look at BranchModifier and case -3. 
This can be a branch modifier that sets .Skip to true and .notes[] to - "skipping because no gcloud command"